modules/clients/mount_wekafs.sh

```diff
@@ -34,14 +34,23 @@ else
 fi
 
 backend_ip="$${ips[RANDOM % $${#ips[@]}]}"
+
+backend_install_url="$backend_ip:14000/dist/v1/install"
```

Comment:
you can just provide in `alb_listener_protocol` the protocol instead of the port and then use it:
`$alb_listener_protocol://$backend_ip:14000/dist/v1/install`